srcpkgs/wren/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'wren'
+pkgname=wren
+version=0.3.0
+revision=1
+build_wrksrc=./projects/make
+build_style=gnu-makefile
+#make_build_args=""
+#make_dirs="/var/log/dir 0755 root root"
+short_desc="Small, class-based, concurrent scripting language - library files"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="MIT"
+homepage="http://wren.io"
+distfiles="https://github.com/wren-lang/wren/archive/${version}.tar.gz"
+checksum=c566422b52a18693f57b15ae4c9459604e426ea64eddb5fbf2844d8781aa4eb7
+
+do_install() {
+	vinstall ./lib/libwren.a 0755 /usr/lib/
+	vinstall ./lib/libwren.so 0755 /usr/lib/
+	vlicense ./LICENSE
+}
+
+wren-devel_package() {
+	short_desc="Small, class-based, concurrent scripting language - source files"
+	depends="${sourcepkg}-${version}_${revision}"
+	pkg_install() {
+		vmkdir /usr/include/wren
+		vinstall "./src/include/wren.h" 0644 /usr/include/wren
+		vinstall "./src/optional/*" 0644 /usr/include/wren
+		vinstall "./src/vm/*" 0644 /usr/include/wren
```

Comment:
Does this work for building other packages? Looks like you could be messing with the expected file structure. Why do we need to ship source files for?

Reply:
the last argument for `vinstall` should be `usr/include/wren`, no leading slash.

Reply:
You're right, I was messing up the file structure, the new commit should fix that. The way `wren` works is that it's designed to be embedded in other programs. So including wren.h and other source files is useful when someone wants to include those in their project.